fees/lista-lisusd/index.ts

```diff
@@ -53,15 +59,53 @@ const fetch = async (options: FetchOptions) => {
     ],
   });
 
-  const logs_exit = await options.getLogs({
+  const eth_transfer2 = await options.getLogs({
+    target: eth,
+    topics: [
+      "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+      "0x000000000000000000000000a230805c28121cc97b348f8209c79bebea3839c0",
+      "0x00000000000000000000000034b504a5cf0ff41f8a480580533b6dda687fa3da",
+    ],
+  });
+
+  // flash loan
+  const lisusd_transfer1 = await options.getLogs({
+    target: lisUSD,
+    topics: [
+      "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+      "0x000000000000000000000000a230805c28121cc97b348f8209c79bebea3839c0",
+      "0x0000000000000000000000008d388136d578dcd791d081c6042284ced6d9b0c6",
+    ],
+  });
+
+  const lisusd_transfer2 = await options.getLogs({
+    target: lisUSD,
+    topics: [
+      "0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef",
+      "0x000000000000000000000000a230805c28121cc97b348f8209c79bebea3839c0",
+      "0x00000000000000000000000034b504a5cf0ff41f8a480580533b6dda687fa3da",
```

Comment:
why double get same event is it double count

Reply:
we have two vault addresses.

Reply:
not it same event and same source it duplicate